lib/portlayer/exec/container_cache.go

```diff
@@ -79,3 +117,109 @@ func (conCache *containerCache) Remove(idOrRef string) {
 	}
 
 }
+
+func isContainerID(id string) bool {
+	// ID should only be lower case chars & numbers
+	match, _ := regexp.Match("^[a-z0-9]*$", []byte(id))
+	return match
+}
+
+// call back for the event stream
+func callme(ie event.Event, sess *session.Session) {
+	// grab the container from the cache
+	container := containers.Container(ie.Reference())
```

Comment:
we could, but the nil check is in the event manager -- the only time we callback is if an event is present